packages/office-addin-manifest/src/commands.ts

```diff
@@ -170,3 +171,17 @@ export async function validate(
     logErrorMessage(err);
   }
 }
+
+export async function exportManifest(command: commander.Command) {
+  try {
+    const outputPath: string | undefined = command.output ?? "./package/manifest.zip";
+    const manifestPath: string | undefined = command.manifest ?? "./manifest.json";
+    const assetsPath: string | undefined = command.assets ?? "./assets";
```

Comment:
Also, why do we need to put this `|undefined`? I think the type is always `string` right?

Reply:
Good point.  Seemed to be the pattern, but unnecessary here.